Inventory.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using InventoryManagementSystem.ProductInfo;
+
+namespace InventoryManagementSystem.InventoryInfo
+{
+    public class Inventory
+    {
+        private readonly int _Id;
+        public static int InventoryIdCounter { get; private set; }
+        public string InventoryName { get; set; }
+        private List<Product> ProductList;
+
+        public Inventory() : this("DefaultInventory")
+        {
+
+        }
+
+        public Inventory(string inventoryName)
+        {
+            InventoryName = inventoryName;
+            ProductList = new List<Product>();
+            _Id = InventoryIdCounter++;
+        }
+
+        public override string ToString() => $"Id {_Id} : Name = {InventoryName}";
+
+        public void AddProduct()
+        {
+            Console.Write("Product name : ");
+            var prodcutName = Console.ReadLine();
```

Comment:
It shouldn't be `Inventory` responsibility to read the input from user. `AddProduct` should accept `Product` object, and the caller/consumer should handle the I/O and get the data from wherever he wants.